overlay.d/05rhcos/usr/lib/dracut/modules.d/40rhcos-fde/coreos-luks-generator

```diff
@@ -17,6 +17,14 @@ add_requires() {
     ln -sf "../${name}" "${requires_dir}/${name}"
 }
 
+# We don't need to mount the local disk if we are in the kdump initramfs
+# and kdump is configured to export the logs to a remote target
+# See https://issues.redhat.com/browse/OCPBUGS-42351
+# Also see https://github.com/coreos/fedora-coreos-config/commit/8ced0db61f23a9b59ef096317fb6fc225bfba133
+if getargbool 0 'kdump_remote_ip'; then
+   exit 0
+fi
```

Comment:
is `kdump_remote_ip` a new karg we are adding here? will we need docs for it? If it's specific to use I'd prefer to add `coreos` to the front like `coreos.kdump_remote_ip`.

Reply:
I am not adding any kargs, I simply look for the one kexec uses.
More details in this commit message : https://github.com/coreos/fedora-coreos-config/commit/8ced0db61f23a9b59ef096317fb6fc225bfba133